spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  describe 'Initialize' do
+    it 'exists' do
+      facility = FacilityFactory.new
+
+      expect(facility).to be_a(FacilityFactory)
+    end
+
+    it 'creates new facilities from OR' do
+      facility = FacilityFactory.new
+      or_dmv_office_locations = DmvDataService.new.or_dmv_office_locations
+      or_facilities = facility.create_facility(or_dmv_office_locations)
+      
+      expect(or_facilities[0].name).to be_a String
+      expect(or_facilities[1].address).to be_a String
+      expect(or_facilities[2].phone).to be_a String
```

Comment:
You can also check that the objects are Facility objects. `expect(or_facilities.first).to be_a Facility`. But yeah, that's really all we can check here since we don't want to check the specific data since we don't control the data.